src/frontend/webpack.config.js

```diff
@@ -31,7 +31,7 @@ module.exports = {
   optimization: {
     minimize: true,
     minimizer: [new TerserPlugin({
-      parallel: true,
+      parallel: process.env.CI ? 2 : true,
```

Comment:
By default `extractComments` is enabled so several text files are created next to built files. You can disable this option:
```suggestion
      parallel: process.env.CI ? 2 : true,
      extractComments: false,
```